src/MixedContentObserver.php

```diff
@@ -2,16 +2,26 @@
 
 namespace Spatie\MixedContentScanner;
 
-use Spatie\Crawler\CrawlObserver;
-use Psr\Http\Message\UriInterface;
-use Psr\Http\Message\ResponseInterface;
 use GuzzleHttp\Exception\RequestException;
+use Psr\Http\Message\ResponseInterface;
+use Psr\Http\Message\UriInterface;
+use Spatie\Crawler\CrawlObserver;
 
 class MixedContentObserver extends CrawlObserver
 {
+    private $crawledCss = [];
+
+    private $shouldExtractLinkedCss = false;
+
     public function crawled(UriInterface $url, ResponseInterface $response, ?UriInterface $foundOnUrl = null)
     {
-        $mixedContent = MixedContentExtractor::extract((string) $response->getBody(), $url);
+        [$mixedContent, $linkedCss] = MixedContentExtractor::extract((string) $response->getBody(), $url);
+
+        if ($this->shouldExtractLinkedCss) {
+            foreach ($linkedCss as $css) {
+                $mixedContent = array_merge((array) $mixedContent, (array) $this->linkedCssFound($css, $url));
```

Comment:
I'm confused by this code. You add something to `$mixedContent`, but judging by the name `$this->linkedCssFound` would return a boolean, `true` is there is a linked css. 

This is probably not what is happening. Could you improve the `linkedCssFound` name?

Reply:
Renamed function to be getMixedContentFromLinkedCss